src/sentry/discover/endpoints/discover_saved_queries.py

```diff
@@ -39,6 +39,9 @@ def get(self, request, organization):
                 if key == "name" or key == "query":
                     value = " ".join(value)
                     queryset = queryset.filter(name__icontains=value)
+                elif key == "version":
+                    value = " ".join(value)
+                    queryset = queryset.filter(version=value)
```

Comment:
I don't think we would want to have `version` as a search query. 

I feel that a `version` query string may be better: `version = request.query_params.get("version")`.

I'll bring this up in the discover Slack channel for a potential further discussion with the team.